haiku-apps/genio/genio-1.0.0~beta.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="The Haiku IDE"
+DESCRIPTION="Genio is an IDE for the Haiku operating system. \
+It has a rich editor supporting syntax highlight, auto completion and other advanced features. \
+It contains a set of templates to quickly develop and build your projects."
+HOMEPAGE="https://github.com/Genio-The-Haiku-IDE/Genio/releases"
+COPYRIGHT="2022-2023 The Genio Team"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+srcGitRev="0655bca6b6851e746db160f949b831955580b337"
+SOURCE_URI="https://github.com/Genio-The-Haiku-IDE/Genio/archive/v1.tar.gz"
+CHECKSUM_SHA256="7ae9a8affd59869741b7a891384fc733ce78fd404ba2e433708a68a473d11358"
+SOURCE_FILENAME="Genio-v1.tar.gz"
+SOURCE_DIR="Genio-1"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	Genio${secondaryArchSuffix} = $portVersion
```

Comment:
Should reflect recipe's name (no capitols):
```diff
-	Genio${secondaryArchSuffix} = $portVersion
+	genio${secondaryArchSuffix} = $portVersion
```

Reply:
Also in the line "app:Genio..." ?

Reply:
No, "app:Genio" is fine :)

Reply:
> No, "app:Genio" is fine :)
> 
> Err, without secondaryArchSuffix

Weird: I copied from scintilla and there they use 
```
lib:libscintilla$secondaryArchSuffix = $portVersion
```

Reply:
lib:libfoo$secondaryArchSuffix are libraries installed into $prefix/lib/x86, those installed in $prefix/lib don't need the suffix.
For binaries placed in $appsDir this is the same, no suffix is needed here.

Reply:
Sorry, so to clean up things, is this correct ?
Also if it only works with the secondary arch x86, and not x86_gcc2 ?

```
genio$ = $portVersion
app:Genio = $portVersion
```

Reply:
We are building for secondary architecture (primary marked broken !x86_gcc2) so:
```
genio$secondaryArchSuffix
app:Genio = $portVersion
```

Reply:
Done.